cli/CHANGELOG.md

```diff
@@ -1,18 +1,18 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.7.3
 
-_Released 4/16/2024 (PENDING)_
-
-**Misc:**
-
-- Updated the Chrome flags to not show the "Enhanced Ad Privacy" dialog. Addresses [#29199](https://github.com/cypress-io/cypress/issues/29199).
-- Suppresses benign warnings that reference Vulkan on GPU-less hosts. Addresses [#29085](https://github.com/cypress-io/cypress/issues/29085). Addressed in [#29278](https://github.com/cypress-io/cypress/pull/29278).
+_Released 4/11/2024_
 
 **Bugfixes:**
 
 - Fixed an issue where asserts with custom messages weren't displaying properly. Fixes [#29167](https://github.com/cypress-io/cypress/issues/29167)
 - Fixed and issue where cypress launch arguments were not being escaped correctly with multiple values inside quotes. Fixes [#27454](https://github.com/cypress-io/cypress/issues/27454).
```

Comment:
This should say

"Fixed an issue where Cypress" ...

Reply:
I should be able to fix the typo retroactively. Thanks for the review @MikeMcC399 !